src/main/kotlin/racingcar/racingcar/Car.kt

```diff
@@ -0,0 +1,32 @@
+package racingcar.racingcar
+
+class Car(
+    private val indicator: Indicator
+) {
+    private var currentLocation: Int = 0
+
+    fun go(turn: Int) {
+        val current = indicator.current(turn)
+        if (canIGo(current)) {
+            forward()
+        }
+    }
+
+    private fun canIGo(number: Int): Boolean {
+        val intRange = AVAILABLE_RANGE
+        return number in intRange
```

Comment:
```suggestion
        return number in AVAILABLE_RANGE
```

상수 이름이 `RANGE이니 합쳐줘도 좋을 것 같아요~

Reply:
오 넵!